src/main/java/frc/robot/commands/elevator/GetMaxHeight.java

```diff
@@ -0,0 +1,45 @@
+package frc.robot.commands.elevator;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.constants.ElevatorConstants;
+import frc.robot.subsystems.Elevator;
+
+public class GetMaxHeight extends CommandBase {
+  Elevator m_elevator; 
+  /**
+   * Calibrate the elevator by resetting the encoder after it hits the bottom
+   * limit switch. 
+   * @param elevator
```

Comment:
add some descriptions for the parameters even if they are pretty much the paramater name